lib/services/service_constants.dart

```diff
@@ -1,2 +1,9 @@
-const String apiBaseUrl = 'http://localhost:8080';
+import 'package:flutter/foundation.dart';
+
+const String androidBaseUrl = 'http://10.0.2.2:8080';
+
+String apiBaseUrl = defaultTargetPlatform == TargetPlatform.android
```

Comment:
At some point we will need to get this from an environment variable or a config of some kind (builds installed on a user's device won't work with local base URLs).

Maybe add this to a TD item?